presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -122,6 +122,7 @@
     private int filterAndProjectMinOutputPageRowCount = 256;
     private int maxGroupingSets = 2048;
     private boolean legacyUnnestArrayRows;
+    private boolean defaultFilterFactorEnabled;
```

Comment:
- This should be on by default.

- move the fields next to other CBO fields (perhaps just after `enableStatsCalculator`).  see https://github.com/prestodb/presto/pull/11861#discussion_r231792760

Reply:
We are still evaluating this. And also this flag enabled without https://github.com/prestodb/presto/pull/11982 doesn't make much sense anyway. Once the evaluation is done - we can enabled it.

Reply:
Agree with @arhimondr.  I'm going to leave this disabled and turn it on by default in a follow on patch.